ufc/bandit-flags-v1.json

```diff
@@ -227,6 +227,31 @@
                 }
             ],
             "totalShards": 10000
+        },
+        "cold_start_bandit": {
```

Comment:
Flag in the UFC so that we can request an assignment from the cold-started bandit